io.github.trigg.discover_overlay.yaml

```
@@ -0,0 +1,16 @@
+app-id: io.github.trigg.discover_overlay
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+command: discover-overlay
+modules: 
+  - python3-discover-overlay.json
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

```suggestion
  - --socket=fallback-x11
  - --socket=wayland
```

if it supports wayland

---

The explanation for fallback led me to believe I wouldn't want it as in some cases the app needs to prefer X11. Am I wrong about that?